subgraph/schema.graphql

```diff
@@ -117,3 +119,14 @@ type Receipt @entity {
   "Increasing number of receipt"
   receiptNumber: BigInt!
 }
+
+type ReferrerFee @entity {
+  "Transaction Hash"
```

Comment:
This is not right? Please update the comment :)

Reply:
Yeah i think that one should be right since i use "Transaction Hash" as `ID!`